README.md

```diff
@@ -1,3 +1,52 @@
-# harvester-prototype
 
-This repository is for the frontend of the project react
+# Harvester Prototype (FertiScan)
+
+## Overview Description
+
+---
+The FertiScan project, initiated by the AI-Lab, aims to streamline the process
+of capturing and organizing information on fertilizer labels. By leveraging AI
+technology, the mobile application will allow users to scan fertilizer labels,
+extract essential data, and automatically load this data into a database for
+easy access and analysis. This project involves the AI-Lab, the CFIA, and
+clients such as inspectors and teams who will benefit from the streamlined
+process. FertiScan is a web application that facilitates the inspection process
+wherever internet access is available. By automating fertilizer bag inspection,
+the project aims to increase the efficiency and accuracy of the inspection
+process. Through the integration of the Gemini Vertex API, the system can
```

Comment:
> Gemini Vertex API

Could you provide a hyperlink reference to the API official doc

Reply:
done it